core/tests/protractor_desktop/additionalEditorFeatures.js

```diff
@@ -383,6 +385,390 @@ describe('Full exploration editor', function() {
     await users.logout();
   });
 
+  it(
+    'should show merge changes when the changes are not conflicting',
+    async function() {
+      await users.createUser(
+        'user9@editor.com',
+        'user9Editor');
+      await users.createUser(
+        'user10@editor.com',
+        'user10Editor');
+      await users.login('user9@editor.com');
+      await workflow.createExploration(true);
+      var explorationId = await general.getExplorationIdFromEditor();
+      await explorationEditorMainTab.setStateName('first card');
+      await explorationEditorPage.navigateToSettingsTab();
+      await explorationEditorSettingsTab.setTitle('Testing lost changes modal');
+      await explorationEditorSettingsTab.setCategory('Algebra');
+      await explorationEditorSettingsTab.setObjective('To assess happiness.');
+      await explorationEditorSettingsTab.openAndClosePreviewSummaryTile();
+      await explorationEditorPage.saveChanges();
+      await workflow.addExplorationManager('user10Editor');
+      await explorationEditorPage.navigateToMainTab();
+
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('How are you feeling?');
+      });
+      await action.waitForAutosave();
+      await users.logout();
+
+      await users.login('user10@editor.com');
+      await general.openEditor(explorationId, true);
+      await explorationEditorPage.navigateToSettingsTab();
+      await explorationEditorSettingsTab.setTitle('Title Changed');
+      await explorationEditorSettingsTab.setObjective('Objective Changed.');
+      await explorationEditorPage.saveChanges();
+      await users.logout();
+
+      await users.login('user9@editor.com');
```

Comment:
Should we expect that no "lost changes" is shown (since merge is handled)?

Reply:
Added this.

Reply:
Done.